src/containers/BooksList.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Book from '../components/Book';
+import './BooksList.css';
+
+const BooksList = (props) => {
+  const { books } = props;
+  return (
+    <table>
+      <thead>
+        <tr>
+          <th>id</th>
+          <th>title</th>
+          <th>category</th>
+        </tr>
+      </thead>
+      <tbody>
+        { books.map((book) => <Book book={book} key={book.id} />)}
+      </tbody>
+    </table>
+  );
+};
+
```

Comment:
I dont see you connecting to redux store to fetch books
https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/BooksList/ConnetToRedux.png

Reply:
use `mapStateToProps` to fetch books from the `store`

Reply:
Thanks Louis, we fixed that.